roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -64,7 +64,7 @@ edpm_bootstrap_network_service: NetworkManager
 # Control resolv.conf management by NetworkManager
 # false = disable NetworkManager resolv.conf update (default)
 # true = enable NetworkManager resolv.conf update
-edpm_bootstrap_network_resolvconf_update: false
+edpm_bootstrap_network_resolvconf_update: "{{ edpm_network_config_nmstate | default(False) }}"
```

Comment:
This will need edpm_network_config_nmstate to be set in nodeset in order to override it. But you also trying to update role default in https://github.com/openstack-k8s-operators/edpm-ansible/pull/908/files and role default change will not work unless nodeset have overrides

Reply:
updating default(True) then will work for all scenarios.

Reply:
No it will not as then will require explicit edpm_network_config_nmstate=false or edpm_bootstrap_network_resolvconf_update=false in already deployed cluster nodeset with ifcfg provider.

Also will break the CI as current CI running with ifcfg provider.

Reply:
yes we need to add in nodeset CR to fix issue, we cannot add default value for all scenarios, this default change handles for nmstate scenarios, for ifcfg we need to update nodeset CR for update scenarios.